src/UI/Buyer/angular.json

```diff
@@ -56,25 +109,27 @@
               ]
             },
             "deploy": {
-              "fileReplacements": [ {
-                "replace": "src/environments/environment.local.ts",
-                "with": "src/environments/environment.deploy.ts"
-              } ],
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.local.ts",
+                  "with": "src/environments/environment.deploy.ts"
+                }
+              ],
               "styles": [
                 {
                   "input": "src/styles/themes/defaultbuyer/_main.scss",
                   "bundleName": "defaultbuyer",
-                  "inject": false
+                  "inject": true
```

Comment:
I think we want inject to be false so we can manually inject at release time and swap out themes by app config